shared/tracker/action.render.desktop.js

```diff
@@ -3,41 +3,59 @@
 import React, {Component} from 'react'
 import {FlatButton} from 'material-ui'
 import commonStyles from '../styles/common'
+import {globalColors} from '../styles/style-guide'
 import {normal, checking, warning} from '../constants/tracker'
+import flags from '../util/feature-flags'
 
-import type {Styled} from '../styles/common'
 import type {ActionProps} from './action.render'
 
 export default class ActionRender extends Component {
-  props: ActionProps & Styled;
```

Comment:
the style prop was unneeded and I think good to try to avoid external styles if unneeded